client/js/store/procedure/NewProcedure.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * (c) 2010-present DEMOS E-Partizipation GmbH.
+ *
+ * This file is part of the package demosplan,
+ * for more information see the license file.
+ *
+ * All rights reserved
+ */
+
+const NewProcedure = {
+  namespaced: true,
+
+  name: 'newProcedure',
+
+  state: {
+    requireField: true
+  },
+
+  mutations: {
+    /**
+     *
+     * @param state {Object}
+     * @param value {Boolean} Sets required form field dynamically to true/false.
+     */
+    setRequiredField (state, value) {
```

Comment:
Where is this used?

Reply:
This mutation is being called in the bimschg addon in `BimschgAntrag.vue` which is loaded via the addon wrapper in `DpNewProcedure.vue`